corehq/apps/app_manager/suite_xml/sections/entries.py

```diff
@@ -971,38 +967,41 @@ def set_id(datum_meta, new_id):
         return ret + datums_remaining
 
     @staticmethod
-    def _get_module_for_persistent_context(detail_module, module_unique_id):
-        module_for_persistent_context = detail_module.get_app().get_module_by_unique_id(module_unique_id)
+    def _get_module_for_persistent_context(app, module_unique_id):
```

Comment:
Optional additional refactor: Looks like you could remove `@staticmethod` and then would have `self.app` available instead of passing in the app.

Reply:
Good idea thanks